src/blockchain/utils.ts

```diff
@@ -407,12 +407,20 @@ export const getTokenSymbol = (symbol: ITokenSymbols) => {
   }
 }
 
-export const getLPTokenSymbol = (symbol: ILPTokenSymbols): IU3LPToken => {
+export const getLPTokenSymbol = (
+  symbol: ILPTokenSymbols,
+  provider: BaseProvider
+): IU3LPToken => {
+  if (!provider) console.warn('getLPTokenSymbol called with no provider')
   switch (symbol) {
     case X_U3LP_A:
       return { 0: DAI, 1: USDC }
     case X_U3LP_B:
-      return { 0: USDC, 1: USDT }
+      if (provider.network.chainId === ChainId.Arbitrum) {
```

Comment:
This isn't the right way to get the `chainId` from `network`
Ref: https://docs.ethers.io/v5/api/providers/#providers--networks
```
const { chainId } = await provider.getNetwork()
```